Frontend-v1-Original/components/options/stake.tsx

```diff
@@ -204,7 +226,21 @@ export function Stake() {
           <div>${formatCurrency(totalStakedValue)}</div>
         </div>
         <div className="flex items-center justify-between">
-          <div>APR</div>
+          <div className="flex items-center justify-start gap-5">
+            <div>APR</div>
+            {displayedRewardTokens && displayedRewardTokens.length > 0 && (
+              <div className="flex items-center justify-center gap-1">
+                {displayedRewardTokens.map((token) => (
+                  <img
+                    key={token.address}
+                    src={token.logoUrl ?? "/tokens/unknown-logo.png"}
+                    alt={`${token.symbol} logo`}
+                    className="h-8 w-8 rounded-full"
```

Comment:
```suggestion
                    className="h-5 w-5 rounded-full"
```
8 is a little big IMO.

<img width="545" alt="image" src="https://github.com/Velocimeter/frontend/assets/126733611/5f906247-e82a-4305-a58d-f0a2b160a7df">

Reply:
So those two icons mean users earn their APR in these two tokens, right?

Reply:
right